src/Routes/RosPage/ros-page.scss

```diff
@@ -85,3 +85,9 @@
     background: var(--pf-global--palette--red-200);
   }
 }
+
+.blue-300{
+  .pf-c-progress__indicator {
+    background: var(--pf-global--palette--blue-300);
+  }
```

Comment:
@kgaikwad I found this blue color in the patternfly docs which I feel is very close to the design shared by Pau. Can you please let me know if this is okay or I should ask him to review the color once?

Reply:
@kgaikwad updated, please refer to my comment [here](https://github.com/RedHatInsights/ros-frontend/pull/60#discussion_r728694860)